### DISCUSSION TITLE
中文变体分清、输入方式清晰

### DISCUSSION THREAD

**Discussion by ghost** [rime/home#1547] [General]
首先第一眼会认为：
拼音输入简体字，注音输入繁体字，
实际情况就比较麻烦了，我囧月拼音默认输入繁体字，然而囧月拼音特地列出了正体选项。所以我该怎么用拼音输入繁体合适？如果说默认输出的繁体是香港版，好歹也按维基百科一样把中文变体分开来吧；
注音倒是没问题，但是为什么有两个输入繁体字的方案？

同时有人反映，"但是因为朙月拼音简化字实际上是繁转简而来，这也是你在查看用户词典时发现所有文字记录都是繁体的原因，不仅如此还会有常用候选字在最后几页才能找到，像“复投”等词库由于繁转简原因不支持词库记忆，于是趁着Rime本次升级直接重新配置了一番" https://myzye.com/posts/86bd16d1/

所以这样不如拼音方案交给大陆人解决，比如四叶草方案
其实也不是说交给大陆人解决，但是现在小狼毫的问题就是中文变体分不清，香港台湾分不清，简体输入方式自带的就好几种，虽然输入法可能解决了简繁一对多的问题，但是没有解决输入方式一对多的问题...

希望按照维基百科模式，重新设计配置，以输入方式分类，分清中文变体。
不然我是搞不清用哪个好。或者参考Gboard（安卓，Google）

**Comment by LEOYoon-Tsaw** [comment]
用戶自己選擇方案、選擇字體、選擇異體字標準，Rime全都支持，沒有哪項不支持的。
